cmd/aries-js-worker/src/aries.js

```diff
@@ -34,6 +34,28 @@ export const Aries = new function() {
         }
     }
 
+    this.Start = async function(opts) {
+        return Aries._invoke("aries", "Start", opts, "timeout while starting aries")
+    }
+
+
+    this.waitForNotification = async function(topic) {
+        return new Promise((resolve, reject) => {
+            const timer = setTimeout(_ => reject(new Error("timed out waiting for notification")), 10000)
```

Comment:
what happens after the timeout?

Reply:
current yield returns with no incoming message. Generator will be called again.

check `startNotifier` in index.html